distribution/docs/src/main/resources/_contents/_endpoints/opensearch-endpoint-contents.adoc

```diff
@@ -0,0 +1,276 @@
+==== OpenSearch Endpoint
+
+The OpenSearch Endpoint enables a client to send query parameters and receive search results. This endpoint uses the input query parameters to create an OpenSearch query. The client does not need to specify all of the query parameters, only the query parameters of interest.
+
+===== Installing the OpenSearch Endpoint
+
+The OpenSearch Endpoint is installed by default with a standard installation in the {ddf-catalog} application.
+
+===== Configuring the OpenSearch Endpoint
+
+The OpenSearch Endpoint has no configurable properties.
+It can only be installed or uninstalled.
+
+===== OpenSearch URL
+
+----
+${secure_url}/services/catalog/query
+----
+
+====== From Code:
+
+The OpenSearch specification defines a file format to describe an OpenSearch endpoint.
```

Comment:
Not sure how useful this section is. Maybe we should consider removing it, or adding more information to make it move useful, similar to what we did for the CometD endpoint?

We should at least add links to the OpenSearch and CDR IPT Specifications.